enginetest/join_planning_tests.go

```diff
@@ -724,6 +724,48 @@ join uv d on d.u = c.x`,
 				types: []plan.JoinType{plan.JoinTypeHash, plan.JoinTypeMerge, plan.JoinTypeLookup},
 				order: []string{"a", "b", "c", "d"},
 			},
+			{
+				q:     "select /*+ LOOKUP_JOIN(xy,applySubq0) */ 1 from xy where x not in (select u from uv)",
```

Comment:
What is this applySubq0 business, how is a customer supposed to discern this magic string, boooooo

Reply:
Unless we support `select * from xy anti join ab` in the parser we have to generate names for scalar subqueries.